composer.json

```diff
@@ -15,8 +15,8 @@
         "source": "https://github.com/woohoolabs/zen"
     },
     "require": {
-        "php": "^8.0.0",
-        "psr/container": "^1.0.0",
+        "php": ">=8.0",
+        "psr/container": "^1.0 || ^2.0",
```

Comment:
The current implementation is not compatible with PSR/Container 1.0 anymore. Only 1.1+ should be supported.